Makefile

```
@@ -125,76 +125,77 @@ dev: dev-install dev-stop
 	  /opt/mesosphere/bin/dcos-net-env console
 
 ##
-## DC/OS E2E
+## miniDC/OS
 ##
 
-DCOS_DOCKER_TRANSPORT ?= docker-exec
-DCOS_DOCKER_CUSTOM_VOLUME ?= "$(BASE_DIR):$(BASE_DIR):rw"
-DCOS_DOCKER_CLUSTER_ID ?= default
-DCOS_DOCKER_MASTERS ?= 1
-DCOS_DOCKER_AGENTS ?= 1
-DCOS_DOCKER_PUBLIC_AGENTS ?= 0
-DCOS_DOCKER_NODE ?= master_0
-DCOS_DOCKER_WEB_PORT ?= 443
-
-dcos-docker-create:
-	@ dcos-docker inspect \
-	      --cluster-id $(DCOS_DOCKER_CLUSTER_ID) \
+MINIDCOS_TRANSPORT ?= docker-exec
+MINIDCOS_CUSTOM_VOLUME ?= "$(BASE_DIR):$(BASE_DIR):rw"
+MINIDCOS_CLUSTER_ID ?= default
+MINIDCOS_MASTERS ?= 1
+MINIDCOS_AGENTS ?= 1
+MINIDCOS_PUBLIC_AGENTS ?= 0
+MINIDCOS_NODE ?= master_0
+MINIDCOS_WEB_PORT ?= 443
+
+minidcos-create:
+	@ minidcos docker inspect \
+	      --cluster-id $(MINIDCOS_CLUSTER_ID) \
 	      > /dev/null 2> /dev/null || \
-	( dcos-docker create dcos_generate_config.sh \
-	      --transport $(DCOS_DOCKER_TRANSPORT) \
-	      --masters $(DCOS_DOCKER_MASTERS) \
-	      --agents $(DCOS_DOCKER_AGENTS) \
-	      --public-agents $(DCOS_DOCKER_PUBLIC_AGENTS) \
-	      --cluster-id $(DCOS_DOCKER_CLUSTER_ID) \
-	      --custom-volume $(DCOS_DOCKER_CUSTOM_VOLUME) \
-	      $(DCOS_DOCKER_OPTS) && \
-	  dcos-docker wait \
-	      --transport $(DCOS_DOCKER_TRANSPORT) \
-	      --cluster-id $(DCOS_DOCKER_CLUSTER_ID) \
+	( minidcos docker create dcos_generate_config.sh \
+	      --transport $(MINIDCOS_TRANSPORT) \
+	      --masters $(MINIDCOS_MASTERS) \
+	      --agents $(MINIDCOS_AGENTS) \
+	      --public-agents $(MINIDCOS_PUBLIC_AGENTS) \
+	      --cluster-id $(MINIDCOS_CLUSTER_ID) \
+	      --custom-volume $(MINIDCOS_CUSTOM_VOLUME) \
+	      $(MINIDCOS_OPTS) && \
+	  minidcos docker wait \
+	      --transport $(MINIDCOS_TRANSPORT) \
+	      --cluster-id $(MINIDCOS_CLUSTER_ID) \
 	      --skip-http-checks )
 
-dcos-docker-destroy:
+minidcos-destroy:
 	@ docker ps \
-	    --filter label=dcos-e2e-web-id=$(DCOS_DOCKER_CLUSTER_ID) \
+	    --filter label=dcos_e2e.cluster_id=$(MINIDCOS_CLUSTER_ID) \
+	    --filter label=dcos_e2e.web_port \
 	    --format '{{.ID}}' \
-	| xargs docker kill > /dev/null
-	@ dcos-docker destroy --cluster-id $(DCOS_DOCKER_CLUSTER_ID)
+	    | while read id; do [ -z "$id" ] || docker kill $id; done > /dev/null
```

```suggestion
	| while read id; do [ -z "$$id" ] || docker kill $$id; done > /dev/null
```

`$i` is not defined in `Makefile` and it always tried to kill container `d`. To avoid the issue, we must follow quoting rules. When I was suggesting the command I checked it in shell, but not in Makefile. My bad.

https://www.gnu.org/software/make/manual/html_node/Variables-in-Recipes.html

Variable and function references in recipes have identical syntax and semantics to references elsewhere in the makefile. They also have the same quoting rules: if you want a dollar sign to appear in your recipe, you must double it (‘$$’). For shells like the default shell, that use dollar signs to introduce variables, it’s important to keep clear in your mind whether the variable you want to reference is a make variable (use a single dollar sign) or a shell variable (use two dollar signs).

---

Oh, yeah I remember that I overlooked that too :/. 

I brought my mac to the office today. I was planning to test everything on Mac myself later today. I am glad you are helping me get this ready, but if you think you are wasting too much time with it, I will gladly make sure all runs smoothly this afternoon.